code/modules/medical/cloning.dm

```diff
@@ -30,6 +30,9 @@
 
 	l_color = "#7BF9FF"
 
+/obj/machinery/cloning/clonepod/empty
```

Comment:
We have 3 valid map PRs open. 2 are either varedits or path replacements (nothing to do with cloning). The third doesn't edit any tiles with clonepods on them.